libs/wingsdk/src/shared/bundling.ts

```diff
@@ -35,8 +43,11 @@ export function createBundle(entrypoint: string, outputDir?: string): Bundle {
       "@winglang/sdk": SDK_PATH,
     },
     minify: false,
+    sourcemap: "external",
     platform: "node",
     target: "node18",
+    write: false,
+    minifyWhitespace: true,
```

Comment:
Unfortunately we can use esbuild plugins (because we use the sync version of the API), so our options of doing this are slim.

I did just think of something to try though: I can go back to the same as before, except instead of deleting the lines and I leave them empty. Sourcemap encodings are sensitive on a per-line basis.

> I wonder if there's any way to leave the comments but instead ensure that they're the same across compilations (i.e. not pointing to temporary directories)?

I forgot that's why we originally removed comments, haha. It's actually possible my other change in this file (using `absWorkingDir`) might've fixed that anyways. I'll check it out.

Reply:
Ok I changed it to not minify, and I confirmed that my other changes got rid of the tmp dir in the path, so now no post-processing is needed on the bundle